source/docs/guide/src/develop_proofs.md

```diff
@@ -1,6 +1,6 @@
 # Using assert and assume to develop proofs
 
-The previous section started with an outline of proof:
+In [an earlier chapter](./spec_lib.md), we started with an outline of proof:
```

Comment:
I think we want "of [a] proof" (I know that was there before you started, but I just noticed it now).